ods_ci/tests/Resources/Files/accelerator/qe_accelerator_profile_2.yaml

```
@@ -0,0 +1,17 @@
+# File to provision an Accelerator Profile instance in test:
+#   "Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI"
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qeacceleratorprofile2
+  namespace: redhat-ods-applications
+spec:
+  description: Modify Accelerator Profile
```

Just an example, but relevant for all yamls:
```suggestion
  description: Test: Modify An Accelerator Profile Using "Accelerator Profiles" Administration UI
```